src/components/atoms/SfRadio/README.md

```diff
@@ -0,0 +1,39 @@
+# SfRadio
+
+Component for simple group of radio buttons, pass an array get selected value via v-model
+
+## Props
+
+- `value` - selected option
+- `options` - array of objects, keys (label) and (value) are required, more keys are optional
+
+## Events
+
+- `input` - event emmited when option is selected
+
+## Slots 
+
+- `content` - slot for your custom html option structure
+- `button` - slot for replacing default checkmark
+
+## CSS Modifiers
```

Comment:
should be docummented as knobs (see other examples)

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/91/commits/0d33e8d6f260893ffb7cb1d968477c6ef880989d